typeql/language/get.feature

```diff
@@ -88,6 +88,28 @@ Feature: TypeQL Get Clause
       match $x isa person; get $y;
       """
 
+  Scenario: Value variables can be specified in a 'get'
+    Given typeql insert
+      """
+      insert
+      $x "Lisa" isa name;
+      $y 16 isa age;
+      $z isa person, has name $x, has age $y, has ref 0;
+      """
+    Given transaction commits
+
+    Given session opens transaction of type: read
+    When get answers of typeql match
+      """
+      match
+        $z isa person, has name $x, has age $y;
+        ?b = 2017 - $y;
+      get $z, $x, ?b;
+      """
+    Then uniquely identify answer concepts
+      | z         | x               | b             |
+      | key:ref:0 | value:name:Lisa | raw:long:2001 |
```

Comment:
Note a huge fan of `raw`. Really we should rename `value` to `attr:name:Lisa` and then use `value` for `b`...
if you don't want to do it now please make an issue to rename `value:name:Lisa -> attr:name:Lisa` throughout, and then rename `raw:_:_` to `value:_:_` throughout. This will be a pretty sweeping change i think! But not very difficult at all.